pkg/lib/cockpit-components-listing-panel.scss

```diff
@@ -0,0 +1,127 @@
+.ct-listing-panel {
```

Comment:
Admittedly I am not sure why you renamed `listing-ct-*` to `ct-listing-*`.
Just curious, was there some reason for that? (of course even if there was none, no reason to change it back :) )

Reply:
This change was from @garrett, afaik he wants to keep all cockpit classes with ct-* prefix from now on.